cmd/verify/verify.go

```diff
@@ -0,0 +1,216 @@
+package verify
+
+import (
+	"errors"
+	"fmt"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	ErrKubectlInstall  = errors.New("kubectl is not installed")
```

Comment:
"not installed" may not be fully accurate. Perhaps change it to "not installed or missing in your $PATH". Same for ErrArgoCD, ErrGit, ErrArlon.

Reply:
Sure will change the error message